plugins/katello/3.15/user_guide/remote_databases/index.md

```diff
@@ -24,6 +24,8 @@ There are two ways to deploy Foreman with remote databases:
     * restore the DBs on remote servers
     * run foreman-installer with right parameters pointing to the databases. It re-configures the databases and start all the services with new DB locations
 
+### Important note: Katello 3.15 changes the default database for pulp from postgres v9.x to postgres v12. External databases should be upgraded to avoid conflicts.
```

Comment:
The correct way to write it is PostgreSQL. I'd also use Pulp.

```suggestion
### Important note: Katello 3.15 changes the default database for Pulp from PostgreSQL v9.x to PostgreSQL v12. External databases should be upgraded to avoid conflicts.
```

It may also be good to note that Foreman itself recommends upgrading to version 12. https://theforeman.org/manuals/2.0/index.html#Headlinefeatures already mentions it. https://theforeman.org/manuals/2.0/index.html#3.1.1SupportedPlatforms also states you must at least run PostgreSQL 10 for Foreman itself.

This also suggests MongoDB is no longer needed for Pulp, but that's not really true.